MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -357,6 +377,12 @@ curl http://${host_ip}:8888/v1/multimodalqna \
     -d '{"messages": "What is the revenue of Nike in 2023?"}'
 ```
 
+```bash
+curl http://${host_ip}:8888/v1/multimodalqna     
```

Comment:
You need the `\` at the end of the lines

Reply:
Weird I thought I committed that